website/www/site/i18n/home/logos/en.yaml

```diff
@@ -11,4 +11,4 @@
 # limitations under the License.
 
 - id: home-logos-title
-  translation: "Supported runners"
\ No newline at end of file
+  translation: "Apache Beam Runs in These Environments"
```

Comment:
Prefer sentence case: "Apache Beam runs in these environments"

Reply:
As discussed on today's call, we keep title case everywhere.